src/app/core-module/course/course.component.css

```diff
@@ -7,6 +7,42 @@
   min-width: 470px;
 }
 
+#course-information {
+  display: inline-block;
+  width: calc(100% - 150px);
+  padding: 10px;
+}
+
+#buttons {
+  display: inline-block;
+  width: 150px;
+  padding: 10px;
+}
+
+#buttons button {
+  margin: 5px;
+}
+
+#top-row {
+  display: inline-flex;
+  width: 100%;
+}
+
+#top-row h4 {
+  width: calc(100% - 200px);
+}
+
+#top-row-time-block {
+  display: inline-flex;
+  float: right;
```

Comment:
It's not a very good idea to use floats. Especially if you're already using flexbox.